lib/ace/ext/papeeria/katex-previewer.coffee

```diff
@@ -69,6 +69,9 @@ define((require, exports, module) ->
 
     constructor: (@editor, @jqEditorContainer, @popoverHandler, @equationRangeHandler, @getFormulaElement) ->
       @contextPreviewExists = false
+      @rangeExists = false
+      @rangeCorrect = false
+      @currentRange = null
```

Comment:
Is there 1.0 correlation between `currentRange` and `rangeExists` values? `currentRange` is defined <=> `rangeExists is true` and vice versa, `currentRange is undefined` <=> `rangeExists is false`